Map Version Edits.R

```diff
@@ -1,3 +1,18 @@
+# Script to generate shapefiles with indicators for corner lots.
+#
+# To run, set the township name variables below for the township you're
+# interested in, then run the script. The script will output shapefiles to the
+# `Base Data/` directory. These files should then be uploaded to the following
+# S3 path:
+#
+# s3://ccao-data-raw-us-east-1/location/corner_lot/year={year}/*.shp
+#
+# At some point we may want to automate this upload, but it's manual for now.
+#
+# Note that the script can take upwards of 24 hours to complete, so you may
+# want to run it as a Background Job in RStudio or via a tmux session on
+# the server to ensure that your connection doesn't get killed.
```

Comment:
I haven't yet tried the Background Job approach in RStudio yet, but it seems like it might be easier than running the job in the foreground and manually refreshing the connection; want to give it a try for your next township export and see if it works OK?

Reply:
Background jobs have worked (and what I use during the workday), but there have been trouble with them timing out when on the VPN.